pkg/network/network.go

```diff
@@ -155,6 +185,26 @@ func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
 		nc.DomainTemplate = dt
 	}
 
+	if autoTLS, ok := configMap.Data[AutoTLSKey]; !ok {
```

Comment:
This all can be replaced by `nc.AutoTLS = strings.ToLower(configMap.Data[AutoTLSKey]) == "enabled"`.

Reply:
Done.